### PR TITLE
chore: refactor params variables to args

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

* It changes the name of some variables from `params` to `args`

## Why is it important?

There is a global variable params, this change avoid conflicts with that variable.

### CODE REVIEW

**Review by cachedout — APPROVED** [review]

**Review by mdelapenya — APPROVED** [review]

File: vars/pipelineApmUI.groovy [hunk 1]
```diff
@@ -28,8 +28,8 @@ void call(Map args = [:]){
      FORCE_COLOR = "2"
      GIT_URL = "git@github.com:elastic/kibana.git"
      ES_GIT_URL = "git@github.com:elastic/elasticsearch.git"
-     TEST_BROWSER_HEADLESS = "${params.TEST_BROWSER_HEADLESS}"
-     TEST_ES_FROM = "${params.TEST_ES_FROM}"
+     TEST_BROWSER_HEADLESS = "${args.TEST_BROWSER_HEADLESS}"
+     TEST_ES_FROM = "${args.TEST_ES_FROM}"
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:32]:
```suggestion
     TEST_BROWSER_HEADLESS = "${params.TEST_BROWSER_HEADLESS}"
     TEST_ES_FROM = "${params.TEST_ES_FROM}"
```

File: vars/pipelineApmUI.groovy [hunk 2]
```diff
@@ -89,7 +89,7 @@ void call(Map args = [:]){
            }
            when {
              beforeAgent true
-             expression { return params.build_oss_ci }
+             expression { return args.build_oss_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:92]:
```suggestion
             expression { return params.build_oss_ci }
```

File: vars/pipelineApmUI.groovy [hunk 3]
```diff
@@ -105,7 +105,7 @@ void call(Map args = [:]){
            }
            when {
              beforeAgent true
-             expression { return params.build_no_oss_ci }
+             expression { return args.build_no_oss_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:108]:
```suggestion
             expression { return params.build_no_oss_ci }
```

File: vars/pipelineApmUI.groovy [hunk 4]
```diff
@@ -124,7 +124,7 @@ void call(Map args = [:]){
        }
        when {
          beforeAgent true
-         expression { return params.intake_ci }
+         expression { return args.intake_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:127]:
```suggestion
         expression { return params.intake_ci }
```

File: vars/pipelineApmUI.groovy [hunk 5]
```diff
@@ -142,7 +142,7 @@ void call(Map args = [:]){
        }
        when {
          beforeAgent true
-         expression { return params.ciGroup_ci }
+         expression { return args.ciGroup_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:145]:
```suggestion
         expression { return params.ciGroup_ci }
```

File: vars/pipelineApmUI.groovy [hunk 6]
```diff
@@ -161,7 +161,7 @@ void call(Map args = [:]){
        }
        when {
          beforeAgent true
-         expression { return params.x_pack_intake_ci }
+         expression { return args.x_pack_intake_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:164]:
```suggestion
         expression { return params.x_pack_intake_ci }
```

File: vars/pipelineApmUI.groovy [hunk 7]
```diff
@@ -181,7 +181,7 @@ void call(Map args = [:]){
        }
        when {
          beforeAgent true
-         expression { return params.x_pack_ciGroup_ci }
+         expression { return args.x_pack_ciGroup_ci }
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:184]:
```suggestion
         expression { return params.x_pack_ciGroup_ci }
```

File: vars/pipelineApmUI.groovy [hunk 8]
```diff
@@ -285,7 +285,7 @@ def installNodeJs(nodeVersion, pakages = null){
 def checkoutES(){
 //  useCache('es-source'){
     dir("${ES_BASE_DIR}"){
-      checkout([$class: 'GitSCM', branches: [[name: "${params.ES_VERSION}"]],
+      checkout([$class: 'GitSCM', branches: [[name: "${args.ES_VERSION}"]],
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:288]:
```suggestion
      checkout([$class: 'GitSCM', branches: [[name: "${params.ES_VERSION}"]],
```

File: vars/pipelineApmUI.groovy [hunk 9]
```diff
@@ -309,7 +309,7 @@ def checkoutES(){
 */
 def checkoutKibana(){
 //  useCache('source'){
-    gitCheckout(basedir: "${BASE_DIR}", branch: params.branch_specifier,
+    gitCheckout(basedir: "${BASE_DIR}", branch: args.branch_specifier,
```
Comment (v1v) [inline vars/pipelineApmUI.groovy:312]:
```suggestion
    gitCheckout(basedir: "${BASE_DIR}", branch: params.branch_specifier,
```

**Review by v1v — CHANGES_REQUESTED** [review]

File: vars/pipelineApmUI.groovy (hunk 9 above)
Comment (kuisathaverat) [reply vars/pipelineApmUI.groovy]: I've deleted this pipeline it is not used.

**Review by v1v — APPROVED** [review]

**Pull request merged**